spec/libraries/aws_engine_spec.rb

```diff
@@ -39,15 +43,16 @@ class DummyMessage
   module S3
     class DummyClient
       def put_object(params = {})
-
       end
+
       def get_object(params = {})
         filename = params[:response_target]
         File.open(filename, 'wb') { |f| f.write('hoge') }
       end
-      def delete_objects(params = {})
 
+      def delete_objects(params = {})
       end
```

Comment:
__[reek]__
* [Attribute] is a writable attribute [:ref](https://github.com/troessner/reek/blob/master/docs/Attribute.md)